pkg/discovery/helper.go

```diff
@@ -29,6 +29,9 @@ import (
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/restmapper"
 
+	api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
our standard is to alias this import as `velerov1api`, rather than just `api`.

Reply:
Done!